katago_server/trainings/models/network.py

```diff
@@ -41,11 +67,16 @@ class Meta:
         verbose_name = _("Network")
         verbose_name_plural = _("Networks")
 
+    # QUESTION (lightvector): Do any changes need to be made such that this can be a different identifier supplied by the training process?
     uuid = UUIDField(_("unique identifier"), default=uuid.uuid4, db_index=True)
     created_at = DateTimeField(_("creation date"), auto_now_add=True)
+    # QUESTION (lightvector): What is parent_network for?
     parent_network = ForeignKey("self", null=True, blank=True, related_name="variants", on_delete=PROTECT)
     # Some description of the network itself
+    # QUESTION (lightvector): Theoretically one day we could have an architecture that is different than a resnet, do we want to make blocks optional?
     nb_blocks = IntegerField(_("number of blocks in network"))
+    # QUESTION (lightvector): Not just theoretically, but actually it is experimentally possible for one to try networks with variable numbers of
+    # channels in different layers of different blocks, like "wide" resnets and such. Do we want to make channels optional?
     nb_channels = IntegerField(_("number of channels in network"))
     model_architecture_details = JSONField(_("network architecture schema"), null=True, blank=True, default=dict)
```

Comment:
I think we should maybe just replace "nb_bloc" and "nb_channel" by a string, eg:

"b20c192"
"b20c[168-256] (wide)"

We loose a bit of UX (user cannot order network by size for instance) but it is simpler and a bit more flexible.